lang/ofc/Portfile

```diff
@@ -9,26 +9,27 @@ PortGroup           makefile 1.0
 # strndup
 legacysupport.newest_darwin_requires_legacy 10
 
-github.setup        CodethinkLabs ofc 1 release/
```

Comment:
can you not just add the patches instead as we would normally do?

Reply:
Ok, sure.

Reply:
@reneeotten Sorry, forgot about this. Done now.